doc/api/net.md

```diff
@@ -961,9 +970,13 @@ added: v0.1.90
 -->
 
 * `port` {number} Port the socket should connect to. Will be passed to
+  <!--lint disable maximum-line-length-->
   [`socket.connect(port[, host][, connectListener])`][`socket.connect(port, host)`].
+  <!--lint enable maximum-line-length-->
```

Comment:
There are like 10+ of these comments inserted in this file alone...

Reply:
The main reason is because ``[`socket.connect(port[, host][, connectListener])`][`socket.connect(port, host)`]`` length is > 80 so, a posible solution was to rename ``[`socket.connect(port, host)`]`` so, it'll be better to skip that paragraph and leave the link variables as they were....